internal/command/release_create.go

```diff
@@ -0,0 +1,181 @@
+package command
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/simplesurance/baur/v3/internal/command/term"
+	"github.com/simplesurance/baur/v3/internal/log"
+	"github.com/simplesurance/baur/v3/pkg/baur"
+	"github.com/simplesurance/baur/v3/pkg/storage"
+
+	"github.com/spf13/cobra"
+)
+
+const releaseCreateExample = `
+baur release create stable		      create a release named 'stable',
+					      including all tasks
+baur release create --include '*.build' v3    create a release named 'v3',
+					      including only tasks called
+					      'build'`
+
+var releaseCreateLongHelp = fmt.Sprintf(`
+Creates a named snapshot of the status of tasks.
+The snapshot includes information about the tasks and all their outputs.
+
+All tasks included in the release must have a run state of Exist.
+Additional information can be stored with the release database.
+Because it is stored in database, this should not consume too much disk space.
+
+By default all tasks are included in the release. This can be narrowed via the
+--include flag. See %s for more information about the TARGET syntax.
+
+Exit Codes:
+  0 - Success
+  1 - Error
+  %d - Release with the same name already exist
+  %d - One or more tasks are in pending state
+`,
+	term.Highlight("baur run --help"),
+	exitCodeAlreadyExist, exitCodeTaskRunIsPending)
+
+type releaseCreateCmd struct {
+	cobra.Command
+	requireCleanGitWorktree bool
+
+	metadataFile string
+	includes     []string
+}
+
+func init() {
+	releaseCmd.AddCommand(&newReleaseCreateCmd().Command)
+}
+
+func newReleaseCreateCmd() *releaseCreateCmd {
+	cmd := releaseCreateCmd{
+		Command: cobra.Command{
+			Use:               "create NAME",
+			Short:             "create a release",
+			Long:              strings.TrimSpace(releaseCreateLongHelp),
+			Args:              cobra.ExactArgs(1),
+			Example:           strings.TrimSpace(releaseCreateExample),
+			ValidArgsFunction: nil, // FIXME: implement completion
```

Comment:
```suggestion
			ValidArgsFunction: nil, // TODO: implement completion
```

Reply:
FIXME actually triggers the CI check to fail, so I won't forget it before merging it,
TODO does not